lib/entities/plans/models.ts

```diff
@@ -0,0 +1,17 @@
+/** @format **/
+
+export type IUsers = {
+  identifier: string;
+  firstName: string;
+  lastName: string;
+  emailAddress: string;
+}
+
+export type IApplications = {
+  identifier: string;
+  planIdentifier: string;
+  name: string;
+  apiKey: string;
+  deploymentTrackingSetup: boolean;
+  users: IUsers[];
+};
```

Comment:
These should not be plural, as they signify a single `Application` or `User`. I believe that these models already exist within the `entities/applications` section.

Reply:
Nice, I added the deploymentTrackingSetup as that's included within the get single app endpoint call too now